### DISCUSSION TITLE
Root

### DISCUSSION THREAD

**Discussion by gchatou21r** [kyujin-cho/pixel-volte-patch#98] [Ideas]
I think it's a nice idea to make it work also with root. So root users could use pixel ims without shizuku. Please consider to add this choice.

**Comment by kyujin-cho** [comment]
Hi! As this app heavily depends on APIs powered by Shizuku I don't it can be done in near future. I'll consider adding root way if there's high demand for such feature. You still can use Shizuku without enabling remote debugging or plugging in the cable if you have root access to the phone.

**Reply by gchatou21r** [reply]
I use zygisk - sui module instead of shizuku app and pixel ims is running perfect now.
